src/Shiny.Notifications/Platforms/Android/NotificationManager.cs

```diff
@@ -229,18 +246,18 @@ protected virtual void CreateChannel(Notification notification)
                 if (!d.IsEmpty())
                     channel.Description = d;
 
-                this.manager.CreateNotificationChannel(channel);
-            }
+                // Set initial sound attributes for channel when it is created
+                if (notification.Sound?.IsCustomSound() ?? false)
```

Comment:
The minlevel check should not be removed

Reply:
The minlevel is already checked at the beginning of the ``CreateChannel`` method so I thought its not necessary below:
(https://github.com/shinyorg/shiny/blob/35e58343af92392d770a8d64f0e9660830bab430/src/Shiny.Notifications/Platforms/Android/NotificationManager.cs#L232